src/educational-technology.rst

```diff
@@ -1,18 +1,183 @@
+as problem solving. This is especially true when the audience is children, who may need more motivation to engage in these forms of learning activities than adults actively seeking
+out education in programming.
+
+.. container:: bib-item
+
+  .. bibliography:: educational-technology.bib
+    :filter: key == 'fessakis2013problem'
+
+  This paper goes over the effectiveness of integrating a programming activity into a regular kindergarten class using puzzles
+  that give movement directions to a lady bug to reach some designated destination. The students in the class overall had fun
+  solving the puzzles. The children solved the problems in a step-wise fasion, typically only giving the lady bug 1-3 move 
+  commands before running the actions and checking to where the lady bug moved.
+
+.. todo:: 
+    Add other game based programming environments
 
 Block-Based Environments
 ========================
 
+Block-based environments are often used for teaching young students (pre-university) and students who have had little or no previous exposure to programming.
+These environments have the advantage that there is a reduced coginitive load on learners when it comes to syntax. Block-based environments can limit or eliminate syntax errors
+which saves novices time and confusion resolving such errors and allows them to focus on the higher level concepts. Additionally, the use of blocks
+minimizes the demand for typing and shows program construction in a graphical way, which is especially helpful for children.
+
+.. container:: bib-item
+
+  .. bibliography:: educational-technology.bib
+    :filter: key == 'wyeth2001electronic'
+
+  Electronic blocks are physical blocks that are designed for young children to familiarize themselves with concepts related to programming (like input and output).
+  There are three types of blocks, sensor (e.g. sound), logic (e.g. not), and action (e.g. light that turns on and off), that can be combined to form stacks. The researchers ran a study with a class of preschoolers. They found that in general
+  the kids seemed to have fun with the blocks. The children had a hard time understanding the logic blocks, but were in general able to understand
+  the sensor and action blocks. The preschoolers had an easier time building complexity between interactions of two stacks as opposed to within a single stack.
+
+.. container:: bib-item
+
+  .. bibliography:: educational-technology.bib
+    :filter: key == 'maloney2010scratch'
+
+  This paper gives an overview of the design of the Scratch programming environment. Scratch is a graphical, block-based programming environment
+  designed for kids who have no programming experience. The system strives to make execution visible and allow for tinkering (learning by trying). The
+  code is live such that any code fragment can be executed simply by clicking on it (no compile-link-run cycle to go through).
+
+.. todo:: 
+    Find evaluation of effectiveness of Scratch
```

Comment:
Finding a paper or two evaluating Scratch would be useful for your second draft

Reply:
here is some work you might refer to here: https://computinged.wordpress.com/2019/08/19/summarizing-findings-about-block-based-programming-in-computing-education/